### DISCUSSION TITLE
Setting the WebUI display field for a relation?

### DISCUSSION THREAD

**Discussion by johnmarianhoffman** [pocketbase/pocketbase#2572] [Q&A] [answered]
Hi All, 

Apologies if this is documented somewhere.  In my searching I couldn't find anything obvious.  I'm also happy to be steered in an entirely different direction if I'm thinking about this incorrectly.

While I work on brining an actual interface online, I'm entering records into PB via the WebUI.  I'd like to be able to set which field is displayed in the WebUI picker when setting a relation field.  For instance, I have a "runs" collection which has a relation to "shoes."  Currently, when adding a run via the WebUI, the shoes are listed by ID (random auto-generated), but I'd rather display the shoe name.  

Is this possible for user-created collections?

The default always seems to be ID for user-created collections, but for the default PB users table, it displays the name/username, so it does at least seem to be possible. 

Thanks in advance!

John

**Comment by ganigeorgiev** [comment, marked as answer]
You can choose the relation display fields from the `relation` field options:

![relation_options](https://i.imgur.com/6EbC4Kh.png)

In future version the "Display fields" will be moved as Collection option to avoid selecting the same fields every time when a `relation` field to that collection is used.

**Reply by johnmarianhoffman** [reply]
Ah of course.  How did I miss that? It makes so much sense! Thank you! Very much appreciate the insight.

**Comment by sudeepgurung** [comment]
Why this functionality have been removed in v0.19.0 ?!

![gone](https://github.com/pocketbase/pocketbase/assets/129731998/de9744d7-07d3-45a7-ae2f-7c87b933db7b)

**Reply by ganigeorgiev** [reply]
> In future version the "Display fields" will be moved as Collection option to avoid selecting the same fields every time when a relation field to that collection is used.

Or in other words, go to your "categories" collection and mark each desired field as "Presentable".

**Reply by austinkeil** [reply]
I've selected each desired field as presentable, but it still seems to show the object ID instead of the fields selected as presentable.

**Reply by ganigeorgiev** [reply]
@austinkeil Without details about your collections structure I'm not sure what could be the cause of your issue but I suspect that it is probably related to https://github.com/pocketbase/pocketbase/issues/3666#issuecomment-2254367329. If you think otherwise, open a new Q&A discussion with export of your collections structure (Admin UI > Settings > Export collections) and I'll try to have a look at it.

_edit: I've just noticed that you've commented in the linked issue. There is no need to spam in multiple places for the same thing._
